<ui version="4.0" >
 <class>MainWindow_BASE</class>
 <widget class="QMainWindow" name="MainWindow_BASE" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>329</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>696</width>
    <height>0</height>
   </size>
  </property>
  <property name="font" >
   <font>
    <pointsize>12</pointsize>
    <stylestrategy>PreferDefault</stylestrategy>
   </font>
  </property>
  <property name="windowTitle" >
   <string>ACE - A COOL Editor</string>
  </property>
  <property name="dockOptions" >
   <set>QMainWindow::AllowNestedDocks|QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QVBoxLayout" >
    <item>
     <widget class="QTabWidget" name="tabWidget_Connections" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Expanding" hsizetype="MinimumExpanding" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize" >
       <size>
        <width>250</width>
        <height>0</height>
       </size>
      </property>
      <property name="tabPosition" >
       <enum>QTabWidget::West</enum>
      </property>
      <property name="currentIndex" >
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab" >
       <attribute name="title" >
        <string>No current connection</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>696</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Connection" >
    <property name="title" >
     <string>&amp;Connection</string>
    </property>
    <addaction name="action_ConnectionOpen" />
    <addaction name="action_ConnectionDisconnect" />
    <addaction name="separator" />
    <addaction name="action_ConnectionProperties" />
    <addaction name="action_ConnectionRefresh" />
    <addaction name="separator" />
    <addaction name="action_ConnectionQuit" />
   </widget>
   <widget class="QMenu" name="menu_Folder" >
    <property name="enabled" >
     <bool>true</bool>
    </property>
    <property name="title" >
     <string>&amp;Folder</string>
    </property>
    <addaction name="action_FolderNew" />
    <addaction name="action_FolderCopy" />
    <addaction name="action_FolderDelete" />
    <addaction name="separator" />
    <addaction name="action_FolderProperties" />
    <addaction name="action_FolderCommit" />
    <addaction name="action_FolderCommitAll" />
    <addaction name="separator" />
    <addaction name="action_FolderFilter" />
   </widget>
   <widget class="QMenu" name="menu_Preferences" >
    <property name="enabled" >
     <bool>false</bool>
    </property>
    <property name="title" >
     <string>&amp;Preferences</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Help" >
    <property name="enabled" >
     <bool>false</bool>
    </property>
    <property name="windowTitle" >
     <string>COOL Editor</string>
    </property>
    <property name="title" >
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout" />
   </widget>
   <widget class="QMenu" name="menu_View" >
    <property name="enabled" >
     <bool>false</bool>
    </property>
    <property name="title" >
     <string>&amp;View</string>
    </property>
   </widget>
   <addaction name="menu_Connection" />
   <addaction name="menu_Folder" />
   <addaction name="menu_Preferences" />
   <addaction name="menu_View" />
   <addaction name="menu_Help" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QToolBar" name="toolBar" >
   <property name="enabled" >
    <bool>true</bool>
   </property>
   <property name="windowTitle" >
    <string>toolBar</string>
   </property>
   <property name="movable" >
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle" >
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable" >
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="action_ConnectionOpen" />
   <addaction name="action_ConnectionRefresh" />
   <addaction name="action_ConnectionDisconnect" />
   <addaction name="separator" />
   <addaction name="action_FolderFilter" />
   <addaction name="action_FolderCommit" />
  </widget>
  <action name="action_ConnectionOpen" >
   <property name="icon" >
    <iconset resource="../icons.qrc" >:/action_icons/icons/network_online.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="action_ConnectionCopy" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>Copy</string>
   </property>
  </action>
  <action name="action_ConnectionProperties" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="../icons.qrc" >:/action_icons/icons/control_panel.png</iconset>
   </property>
   <property name="text" >
    <string>Properties</string>
   </property>
   <property name="visible" >
    <bool>false</bool>
   </property>
  </action>
  <action name="action_ConnectionDisconnect" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="../icons.qrc" >:/action_icons/icons/network_offline.png</iconset>
   </property>
   <property name="text" >
    <string>Disconnect</string>
   </property>
   <property name="visible" >
    <bool>true</bool>
   </property>
  </action>
  <action name="action_ConnectionRefresh" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="../icons.qrc" >:/action_icons/icons/exec.png</iconset>
   </property>
   <property name="text" >
    <string>Refresh</string>
   </property>
   <property name="visible" >
    <bool>false</bool>
   </property>
  </action>
  <action name="action_ConnectionQuit" >
   <property name="icon" >
    <iconset resource="../icons.qrc" >:/action_icons/icons/button_cancel.png</iconset>
   </property>
   <property name="text" >
    <string>Quit</string>
   </property>
  </action>
  <action name="action_FolderNew" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="../icons.qrc" >:/action_icons/icons/folder_edit.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;New</string>
   </property>
   <property name="visible" >
    <bool>false</bool>
   </property>
  </action>
  <action name="action_FolderCopy" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="../icons.qrc" >:/action_icons/icons/folder_documents.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Copy</string>
   </property>
   <property name="visible" >
    <bool>false</bool>
   </property>
  </action>
  <action name="action_FolderDelete" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="../icons.qrc" >:/action_icons/icons/trash_full.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Delete</string>
   </property>
   <property name="visible" >
    <bool>false</bool>
   </property>
  </action>
  <action name="action_FolderProperties" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="../icons.qrc" >:/action_icons/icons/folder_properties.png</iconset>
   </property>
   <property name="text" >
    <string>Properties</string>
   </property>
   <property name="visible" >
    <bool>false</bool>
   </property>
  </action>
  <action name="action_FolderFilter" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset resource="../icons.qrc" >:/action_icons/icons/folder_app2.png</iconset>
   </property>
   <property name="text" >
    <string>F&amp;ilter</string>
   </property>
   <property name="toolTip" >
    <string>Toggle most recent entries</string>
   </property>
   <property name="visible" >
    <bool>true</bool>
   </property>
  </action>
  <action name="action_FolderCommit" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="../icons.qrc" >:/action_icons/icons/button_ok.png</iconset>
   </property>
   <property name="text" >
    <string>Commit</string>
   </property>
  </action>
  <action name="action_FolderCommitAll" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>Commit All</string>
   </property>
   <property name="visible" >
    <bool>true</bool>
   </property>
  </action>
  <action name="actionAbout" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../icons.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>action_ConnectionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow_BASE</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>347</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
